lib/package.json

```diff
@@ -99,18 +99,15 @@
     "@types/react": "^16.9.35",
     "@types/react-dom": "^16.9.8",
     "@types/react-transition-group": "^4.2.4",
-    "@typescript-eslint/eslint-plugin": "^1.6.0",
-    "@typescript-eslint/parser": "^1.6.0",
```

Comment:
These are used in the `.eslintrc` in the root. Therefore they should be declared in the root package.json which was already the case. 1.x was unused but  keeping them around can be dangerous with a lax package manger.